aten/src/ATen/core/PythonFallbackKernel.cpp

```diff
@@ -62,9 +62,13 @@ void pythonFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
   StashTLSOnEntryGuard stash_guard;
 
   // If Python Mode is active, use its PyInterpreter for dispatch
-  const auto& maybe_python_mode_state = at::impl::PythonModeTLS::get_state();
+  // need a copy, not a reference because we will set the state to none before
```

Comment:
What exactly do you mean by "update enable_python_mode to match enable_torch_function_mode"? Some more context in the PR body would help